### ISSUE TITLE
Accessibility of player

### ISSUE THREAD

**Issue by gdkraus** [videojs/video.js#291]
I've used Video JS on some local projects but have modified it some to make the player more accessible to people with disabilities. I had made these changes on the older version of the player and the new one you've recently uploaded requires me to go back and do some manual merging before I can submit it. I'll submit the patches I've made shortly. The fixes include:
1. Making it more keyboard accessible. Currently keyboard-only users lose where the keyboard focus is. Also, when the player controls hide themselves, keyboard-only users cannot access the controls. It requires a mouse to get the controls to appear.
2. Making it more screen reader usable. Some of the buttons change text and screen reader users are not notified of this. Also, ARIA attributes that had been previously added to the scroll bar aren't actually doing anything. I've added in the functionality to make them work for screen reader users so they know what they are doing when the move the time and volume sliders.
3. Making it more usable to speech-only users. Speech recognition users (i.e. Dragon Naturally Speaking) cannot interact with the controls. I've added in functionality that allows them to control the player.
4. Larger captions when in full screen mode. Currently, the captions are quite small in full screen mode. I added a function to make them larger in full screen mode.
